newIDE/app/src/UI/Layout.js

```diff
@@ -27,7 +27,7 @@ const textFieldWithButtonLayoutStyles = {
   container: {
     flex: 1,
     display: 'flex',
-    alignItems: 'flex-start', // Align from the top to stay at the same position when error/multiline
+    alignItems: 'center', // Align from the top to stay at the same position when error/multiline
```

Comment:
Can you verify the impact on this kind of multiline text field:

![image](https://user-images.githubusercontent.com/1280130/92996801-9072a080-f50e-11ea-8098-f29498b8db44.png)

The idea was for the "123" button to *not* move when you add multiple lines. I don't have strong feelings about this, just be sure to remove/update the comment ("Align from the top to stay at the same position when error/multiline") if it's not the case anymore.

Reply:
I can fix the position of the button with the style override:

 ```
MuiButtonBase: {
      // Remove the web-ish "pointer" (hand) cursor from buttons
      root: {
        cursor: 'default',
        alignSelf: "flex-start",
      },
    },
```

Do you think this can have a incidence somewhere else ?

Reply:
I think have a solution for this.
This "center" was a fix for align the font resource selector in the text object, but i can achieve to align with another way.
Return to the original "flex-start", and instead use an override MuiAutocomplete used by the resource selector.

The result stay the same as the previous screenshot and the blue button stay at the start :)

Reply:
> Return to the original "flex-start", and instead use an override MuiAutocomplete used by the resource selector.
> The result stay the same as my previous screenshots and the blue button stay at the start :)

Cool! You can also consider putting an inline style (`style={...}`) in the ResourcesSelector if you want to be 100% sure your custom style won't break anything else.